Plugin/View/LayoutPlugin.php

```diff
@@ -70,13 +70,16 @@ public function __construct(
      * @param  $blockId
      * @return string
      */
-    protected function injectHtmlAttribute($html, $blockId)
+    protected function injectHtmlAttribute($html, $blockId, $stack)
     {
         if (!$html || !$this->config->canInjectCode()) {
             return $html;
         }
 
-        $html = '<!-- MSPDEVTOOLS[' . $blockId . '] -->' . $html . '<!-- /MSPDEVTOOLS[' . $blockId . '] -->';
+
+        $html = '<!-- START[' . $blockId . '] - ' . end($stack) . ' -->' . $html . '<!-- /END[' . $blockId . '] - ' . end($stack) . ' -->';
```

Comment:
It is a good idea, but I think we should use something more specific like "MSPDEVTOOLS_START" and "MSPDEVTOOLS_END". Just to make sure none is using the same namespace in the code.